app/models/obituary.rb

```diff
@@ -5,6 +5,6 @@ class Obituary < ApplicationRecord
 
   private
   def default_image
-    self.image_url ||= 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg'
+    self.image_url ||= '/thumbnail_logo.png'
```

Comment:
Thumbnail logo (just the newspaper) is now our default image for obituaries!

Reply:
Great idea!

Reply:
Love it!

Reply:
Perfect